sql-cli/tests/test_connections.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+
+from sql_cli.connections import _load_yaml_connections, validate_connections
+
+
+def test_validate_connections(caplog):
+    validate_connections()
+
+    postgres_conn_id = "postgres_conn"
+    postgres_conn_id_formatted_string = postgres_conn_id + " " * (25 - len(postgres_conn_id))
```

Comment:
This looks a bit strange tbh. Why does the length of the conn_id matter? Where does the magical `25` come from? :D

Reply:
So, for a prettier output I'm setting the connection ID string width to 25. So that the results appears in a columnar fashion. I have moved this to a constant now and reusing the constant here to fill up spaced for connection IDs having smaller lengths.